src/css/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: "Roboto", Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  background-color: var(--color-white);
+  border: 1px solid var(--border-color);
+  color: var(--color-dark);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
```

Comment:
try to use margin top or bottom only
also use padding in parent block to set indents
<img width="704" alt="CleanShot 2022-08-21 at 16 20 01@2x" src="https://user-images.githubusercontent.com/103505670/185792838-257ede3a-0e90-4a5c-bbb6-b57f8865b76b.png">